src/ansys/mapdl/core/_commands/solution/ocean.py

```diff
@@ -22,10 +22,27 @@
 
 
 class Ocean:
-    def ocdata(self, val1="", val2="", val3="", val14="", **kwargs):
+    def ocdata(
+        self,
+        val1: str = "",
+        val2: str = "",
+        val3: str = "",
+        val4: str = "",
+        val5: str = "",
+        val6: str = "",
+        val7: str = "",
+        val8: str = "",
+        val9: str = "",
+        val10: str = "",
+        val11: str = "",
+        val12: str = "",
+        val13: str = "",
+        val14: str = "",
```

Comment:
Why not use all these arguments in the `mapdl.run` ?

Reply:
Omg...